src/screens/list/UserList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Navbar from "../../components/Navbar";
+import List from "../../components/Navbar";
+
+const UserList = (props) => {
+  return (
+    <div>
+      <Navbar />
```

Comment:
This component will be mapped 
Do you want the navbar above every user

Reply:
no sir this is not mapped